server/pkg/integration/webhook_trigger.go

```diff
@@ -0,0 +1,11 @@
+package integration
+
+type WebhookTrigger struct {
+	onItemCreate    bool
+	onItemUpdate    bool
+	onItemDelete    bool
+	onAssetUpload   bool
+	onAssetDeleted  bool
+	onItemPublish   bool
+	onItemUnPublish bool
```

Comment:
```suggestion
	OnItemCreate    bool
	OnItemUpdate    bool
	OnItemDelete    bool
	OnAssetUpload   bool
	OnAssetDeleted  bool
	OnItemPublish   bool
	OnItemUnPublish bool
```

It's a simple struct, so write it in webhook.go. It may be difficult to see if you add too many files.